components/sceneutil/unrefqueue.cpp

```diff
@@ -1,28 +1,15 @@
 #include "unrefqueue.hpp"
 
-#include <deque>
-
 //#include <osg/Timer>
 
 //#include <components/debug/debuglog.hpp>
```

Comment:
The commented includes are unnecessary here too.

Reply:
I noticed, but zinnschlag commented

> if you are already touching lines of code that need cleanup, then you should do the cleanup.

Since I wasn't already touching those lines I left them where they are

Reply:
I can remove those too if it's blocking, I guess?

Reply:
It is not blocking since you did not touch these lines.